imports/pages/groups/finder/Result.js

```diff
@@ -205,6 +210,36 @@ const GROUP_FINDER_QUERY = gql`
   }
 `;
 
+const getDay = (schedule: String) => {
+  let day;
+  switch (schedule) {
+    case "sunday":
+      day = 0;
```

Comment:
For these, we could just `return 0;` instead of setting the day variable. It'd keep you from having to use break and the variable declaration. 

See https://stackoverflow.com/questions/6114210/in-javascript-is-returning-out-of-a-switch-statement-considered-a-better-practi